javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var myArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] !== typeof 'string') {
```

Comment:
So for this one.
`typeof values[i]` will give you one of these: `string` or `boolean` or `object` or `number`
And this `typeof 'string'` will always be `string`

So essentially you can just do `if (typeof values[i] !== 'string')`